incubator/gogs/Chart.yaml

```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 description: 'Gogs: Go Git Service'
 name: gogs
-version: 0.7.4
-appVersion: 0.11.66
+
```

Comment:
I found most of the chart do not have such blank line for their Chart.yaml, do we need to keep this?

Reply:
@gyliu513 with do you want to remove version and appversion ?
If the chart.yaml doesn't have blank line circle ci is woriied ;-)and mark test failed

Reply:
sorry I test it after Ok chart.yaml accept to have no empty line at the end of file unlike value.yaml
I note that for the next time